include/oneapi/dpl/pstl/hetero/numeric_impl_hetero.h

```diff
@@ -23,12 +23,23 @@
 #    include "algorithm_impl_hetero.h" // to use __pattern_walk2_brick
 #    include "dpcpp/parallel_backend_sycl_utils.h"
 #    include "dpcpp/unseq_backend_sycl.h"
+#    include "dpcpp/parallel_backend_sycl_utils.h"
```

Comment:
The second include of "dpcpp/parallel_backend_sycl_utils.h",  see 24 line.

Reply:
Thanks, I removed it.